src/main/java/md/miller1995/Dealership/controllers/CarController.java

```diff
@@ -22,11 +21,19 @@ public CarController(CarService carService){
         this.carService = carService;
     }
 
-    @GetMapping("/{id}")
+    @GetMapping("/search/by/{id}")
```

Comment:
We should not have verbs on endpoints. Please consider the fact that URL should point to a specific `resource` that we want to create/read/update/delete. More than that, we should go for the most simple way to express our endpoint. In our case, we can have:

```
POST .../api/cars (create a car)
GET .../api/cars (retrieve all cars)
GET .../api/cars/{id} (retrieve a car by id; this will be suggested by the name of the controller method, and we also can use @ApiOperation(value = "Return a car by id") annotation from Swagger to express our intention with the API
```

Reply:
- updated;